pkg/nodeconfig/nodeconfig.go

```diff
@@ -111,6 +114,21 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 	return host.Status.APIServerInternalURL, nil
 }
 
+// getClusterDNSIP returns the IP address of the DNS server, given the service CIDR of the cluster
```

Comment:
I suggest using `clusterDNS` everywhere. A DNS is always an IP address. Plus in some places you say `dnsClusterIP` and in others `clusterDNSIP`. Lets standardize to `clusterDNS` everywhere including commit and PR messages.

Reply:
Changed. Went with `ClusterDNSIP` to explicitly identify the field with a single IP address, given the kubelet's [clusterDNS](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) parameter is a list of IP addresses (`[]string`)